langserve/server.py

```diff
@@ -114,6 +114,13 @@ def _unpack_request_config(
         else:
             raise TypeError(f"Expected a string, dict or BaseModel got {type(config)}")
     config = merge_configs(*config_dicts)
+    if "configurable" in config and config["configurable"]:
+        if "configurable" not in keys:
+            raise HTTPException(
+                422,
+                "Server code has modified the default accepted config keys to "
```

Comment:
nit - should we say something like:

`setting `configurable` fields is disallowed by the server`